src/js/pages/Hukamnama/Layout.js

```diff
@@ -13,16 +13,27 @@ export default class Layout extends React.PureComponent {
   };
   render() {
     const { data } = this.props;
+    const { shabads } = data;
+    const { date, month, year } = data.date.gregorian;
+    const dateString = date + '-' + month + '-' + year;
+    let totalVerses = [];
+
+    shabads.forEach(s => {
+      totalVerses = totalVerses.concat(s.verses);
+    });
```

Comment:
`const totalVerses = shabads.map(s => s.verses)` ?

Reply:
This creates a separate array for verses of each Shabad.
With concat, we get one array with all verses.